### ISSUE TITLE
Dropdown without "label" set results in NVDA not reading currently selected option

### ISSUE THREAD

**Issue by dawolff-ms** [microsoft/fluentui#17532]
### Environment Information

- **Package version(s)**: ^7.165.1
- **Browser and OS versions**: Chrome 89.0.4389.90, Windows 10

### Describe the issue:

If a Dropdown component isn't given a "label" prop, NVDA refuses to announce which option has been selected in the dropdown. Providing an aria-label or aria-labelledby does not fix the issue. 

### Please provide a reproduction of the issue in a codepen:
[Codepen repro](https://codepen.io/wolffd/pen/poRvOZr). Works with Narrator. Doesn't work with NVDA unless label is provided.

#### Actual behavior:
NVDA does not announce the currently selected option unless the developer provides the "label" prop to the Dropdown component.

#### Expected behavior:
NVDA announces the currently selected option no matter how a label is provided for the dropdown (by label, aria-label, or aria-labelledby).

### Documentation describing expected behavior
N/A

**Comment by gouttierre** [comment]
@dawolff-ms - Thank you for filing this item. I see NDVA doesn't cite the actual option (label) - but is will cite 'selected' if an option was pre-selected. 

@khmakoto - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc. @smhigley
